### DISCUSSION TITLE
我想知道 layui.dev 的文档是怎么通过该仓库的 docs中的makedown  生成的

### DISCUSSION THREAD

**Discussion by idcpj** [layui/layui#2331] [讨论]
### 议题条件

- [X] 我确认已查看官方使用文档：https://layui.dev ，但没有找到相关解决方案。
- [X] 我确认已在 [Issues](https://github.com/layui/layui/issues?q=is%3Aissue ) 中搜索过类似的问题，但没有找到相关解决方案。
- [X] 我已仔细阅读： 🍀[ Layui Issue 贡献指南](https://github.com/layui/layui/blob/main/CONTRIBUTING.md)

### 议题类型

功能请求

### 使用版本

v2.9.18

### 问题描述

我想在内网中查询文档,想通过 仓库中的docs  生成一份文档,该如何使用

### 业务代码

```auto
无
```

### 浏览器

无

### 友好承诺

- [X] 我承诺将本着相互尊重、理解和友善的态度进行交流，共同维护 Layui 良好的社区氛围。

**Comment by sentsim** [comment]
Node.js + Markdown 解析器 + laytpl

**Reply by idcpj** [reply]
有可以运行生成文档的脚本吗
